src/controllers/playback/video/index.js

```diff
@@ -354,7 +365,7 @@ export default function (view) {
             toggleSubtitleSync('hide');
 
             // Firefox does not blur by itself
-            if (document.activeElement) {
+            if (document.activeElement && !focusManager.isCurrentlyFocusable(document.activeElement)) {
```

Comment:
This (blur) is probably no longer necessary in modern Firefox.

We cannot use `isCurrentlyFocusable` until the OSD is completely hidden. As an option (not to break something):
```suggestion
            if (osdBottomElement.contains(document.activeElement) || headerElement.contains(document.activeElement)) {
```